app/components/breadcrumbs_component.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Component to render breadcrumbs
+class BreadcrumbsComponent < ViewComponent::Base
```

Comment:
This and its template should probably go in `app/components/spotlight` and be wrapped in `module Spotlight` like all the other spotlight components.

Reply:
Add a test for the component?